plugins/importexport/native/PKPNativeImportExportPlugin.inc.php

```diff
@@ -0,0 +1,338 @@
+<?php
+
+/**
+ * @file plugins/importexport/native/PKPNativeImportExportPlugin.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class PKPNativeImportExportPlugin
+ * @ingroup plugins_importexport_native
+ *
+ * @brief Native XML import/export plugin
+ */
+
+import('lib.pkp.classes.plugins.ImportExportPlugin');
+import('lib.pkp.plugins.importexport.native.PKPNativeImportExportCLIDeployment');
+import('lib.pkp.plugins.importexport.native.PKPNativeImportExportCLIToolKit');
+
+define('ASSOC_TYPE_NONE', -1000);
```

Comment:
Could you move this into PKPApplication with the other assoc types?

Reply:
Fixed